src/pages/data-visualization/gantt-charts/index.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Gantt charts
+description: Complex charts are a powerful way to display complex data sets.
+---
+
+<PageDescription>
+
+A Gantt chart is a project management tool that illustrates work completed over
+a period of time in relation to the time planned for the work. It typically
+includes two sections: the left side outlines a list of tasks, while the right
+side has a timeline with schedule bars that visualize work.
+
+</PageDescription>
+
+<InlineNotification>
+
+**Note:** The charts below are not included in the carbon-charts library. To see
+our roadmap, make feature requests, or contribute, please go to the
+[carbon-charts repository](https://github.com/carbon-design-system/carbon-charts).
+
+</InlineNotification>
+
+## Resources
+
+<Row className="resource-card-group">
+  <Column colLg={4} colMd={4} noGutterSm>
+    <ResourceCard
+      subTitle="Figma kit"
```

Comment:
Typically, we put the title of the kit in the card name. And the Figma icon denotes its a figma file

```suggestion
      subTitle=Gantt Chart Kit (Alpha)"
```

Reply:
thanks, addressed!